packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -270,16 +271,36 @@ class SkiaGoldClient {
         for(Map<String, dynamic> ignore in ignores) {
           final List<String> ignoredQueries = ignore['query'].split('&');
           final String ignoredPullRequest = ignore['note'].split('/').last;
-          if (ignoredQueries.contains('name=$testName') &&
-            ignoredPullRequest == pullRequest) {
-            ignoreIsActive = true;
-            break;
+          final DateTime expiration = DateTime.parse(ignore['expires']);
+          // The currently failing test is in the process of modification.
+          if (ignoredQueries.contains('name=$testName')) {
+            if (expiration.isAfter(DateTime.now())) {
+              ignoreIsActive = true;
+              break;
```

Comment:
The cases where ignores is ["expired ignore", "unexpired ignore"] and ["unexpired ignore", "expired ignore"] should have the same behavior?

Reply:
Removing the break and adding tests. :)